index.scss

```diff
@@ -0,0 +1,201 @@
+$bg-color: hsl(204, 43%, 93%);
+$while-color: white;
+$blue-color: #2ab2af;
+$yellow-color: #c0df34;
+$light-grey-color: #96a6bd;
+$light-blue-color: #2fc6c4;
+$extra-light-blue-color: #e5eff5;
+
+body {
+  background-color: $bg-color;
+  height: 100vh;
+  font-family: Karla;
+  margin: 0;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.main-div-align-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.main-div {
+  max-width: 636px;
+  border-radius: 16px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-shadow: 0px 0px 20px rgb(0 0 0 / 20%);
+}
+
+.intro-section {
+  justify-content: center;
+  background-color: $while-color;
+  padding: 0 40px;
+  height: 216px;
+
+  h1 {
+    margin-top: 38px;
+    font-size: 24px;
+    color: $blue-color;
+  }
+
+  h2 {
+    color: $yellow-color;
+    font-size: 18px;
+    margin: 24px 0px 0px;
+  }
+
+  p {
+    color: $light-grey-color;
+    font-size: 15px;
+    margin: 10px 0px 0px;
+    line-height: 26px;
+  }
+}
+
+.sub-part {
+  display: flex;
+  flex-direction: row;
+}
+
+.description-section {
+  display: flex;
+  flex-direction: column;
+  background-color: $blue-color;
+  flex: 1;
+  padding: 40px;
+
+  h1 {
+    font-size: 18px;
+    color: $extra-light-blue-color;
+    margin: 0;
+  }
+
+  div {
```

Comment:
adding a nested div is not good unless its necessory, just add a classname
e.g. Sometimes we need to pick the first div, we then do: `>div`

Reply:
Also as this div has more nesting, just add a classname and get this div styling out of `description-section`